pallets/registrar/src/lib.rs

```diff
@@ -175,6 +176,8 @@ pub mod pallet {
 
         type RegistrarHooks: RegistrarHooks;
 
+        type InnerRegistrar: RegistrarHandler<Self::AccountId>;
```

Comment:
comment

Reply:
fixed in 9a76e31422088f5a1231e8671063cb0ed3da1581.